src/modules/FileLocksmith/FileLocksmithContextMenu/FileLocksmithContextMenu.vcxproj

```diff
@@ -80,8 +81,8 @@ MakeAppx.exe pack /d . /p $(OutDir)FileLocksmithContextMenuPackage.msix /nv</Com
       <ModuleDefinitionFile>Source.def</ModuleDefinitionFile>
     </Link>
     <PreBuildEvent>
-      <Command>del $(OutDir)\FileLocksmithContextMenuPackage.msix /q
-MakeAppx.exe pack /d . /p $(OutDir)FileLocksmithContextMenuPackage.msix /nv</Command>
+      <Command>del $(OutDir)\*FileLocksmithContextMenuPackage.msix /q
```

Comment:
Outdir is defined to end with `\`. So this path here combines to `...\\*FileLocksmith....`.

Same for the other ones.